server/shared/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -361,7 +361,7 @@ sealed class CORSPolicy(
       }
       def headers = allowHeaders match {
         case AllowHeaders.All => Nil
-        case AllowHeaders.In(_) | AllowHeaders.Reflect =>
+        case AllowHeaders.In(_) | AllowHeaders.Reflect | AllowHeaders.Static(_) =>
```

Comment:
In this case, I don't think the response varies by `Access-Control-Request-Headers`. It's better to have false positives than false negatives here, but it can hurt cache performance.